src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {

```

Comment:
The class name 'ColorSupplier' suggests that it is supplying colors which is fine, but ensure that the class name is not confused with a verb implying action, as per the checklist.

Reply:
how do i fix this problem?